### PR TITLE
Create change files for changes since the last release

### PR DESCRIPTION
**Pull request by anomiex**
#### Changes proposed in this Pull Request:

For everything other than the Jetpack plugin, I created a change file
for any commit touching the project that didn't already have one.

For Jetpack, I did a final pass through the "Needs Changelog" tag and
added a change file for each PR found.

#### Jetpack product discussion

There's probably a ticket somewhere.

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proofread the files.

### CODE REVIEW

File: projects/plugins/jetpack/changelog/backfill-pr-18497 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+API Nonces: Improve performances.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-18497:4]:
```suggestion
Comment: API Nonces: Improve performances. I don't think these changes need a changelog entry.

```

File: projects/plugins/jetpack/changelog/backfill-pr-18765 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Sync: Catch serialization errors.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-18765:4]:
```suggestion
Type: compat.
Comment: Sync: Catch serialization errors.

Synchronization: avoid issues with themes or plugins using anonymous functions within hooks.
```

File: projects/plugins/jetpack/changelog/backfill-pr-18835 [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+Conversation: Handle speaker-name changes propagation closely.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-18835:4]:
```suggestion
Type: compat
Comment: Conversation: Handle speaker-name changes propagation closely. Not worth a changelog entry, the feature isn't yet widely available.

```

File: projects/plugins/jetpack/changelog/backfill-pr-18837 [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: enhancement
+
+Using Jetpack without connecting to WP.com account.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-18837:4]:
```suggestion
Type: enhancement

General: Allow the use of some Jetpack features without connecting to a WordPress.com account.
```

File: projects/plugins/jetpack/changelog/backfill-pr-18844 [hunk 5]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+Dialogue: Improve speakers autocomplete handling.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-18844:4]:
```suggestion
Comment: Dialogue: Improve speakers autocomplete handling. No need for a changelog, the block isn't available yet.

```

File: projects/plugins/jetpack/changelog/backfill-pr-18916 [hunk 6]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+General: Update colors to match upcoming WP 5.7 color changes.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-18916:4]:
```suggestion

General: Update colors to match dashboard color changes in WordPress 5.7.
```

File: projects/plugins/jetpack/changelog/backfill-pr-18959 [hunk 7]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+Nova Restaurant: Fix inline quick editing.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-18959:4]:
```suggestion
Type: bugfix

Custom Content Types: fix inline quick editing of Restaurant menu items.
```

File: projects/plugins/jetpack/changelog/backfill-pr-18981 [hunk 8]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+AMP: Fix compatibility with Podcast Player block in the Customizer preview.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-18981:4]:
```suggestion
Type: compat

Podcast Player Block: Fix compatibility with the AMP plugin in the Customizer preview.
```

File: projects/plugins/jetpack/changelog/backfill-pr-19005 [hunk 9]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Conversation: Support uploading/parsing/composing transcript.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-19005:4]:
```suggestion
Comment: Conversation: Support uploading/parsing/composing transcript. No changelog needed, the block isn't available yet.

```

File: projects/plugins/jetpack/changelog/backfill-pr-19015 [hunk 10]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-19015:2]:
```suggestion
Type: enhancement
```

File: projects/plugins/jetpack/changelog/backfill-pr-19022 [hunk 11]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-19022:2]:
```suggestion
Type: bugfix
```

File: projects/plugins/jetpack/changelog/backfill-pr-19034 [hunk 12]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+Instagram block: Improve compatibility with Gutenberg 10.1.0.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-19034:4]:
```suggestion
Latest Instagram Posts Block: Improve compatibility with Gutenberg 10.1.0.
```

File: projects/plugins/jetpack/changelog/backfill-pr-19036 [hunk 13]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+Podcast Player: Improve compatibility with Gutenberg 10.1.0.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-19036:4]:
```suggestion
Podcast Player Block: Improve compatibility with Gutenberg 10.1.0.
```

File: projects/plugins/jetpack/changelog/backfill-pr-19043 [hunk 14]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+Opentable block: Improve compatibility with Gutenberg 10.1.0.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-19043:4]:
```suggestion
Opentable Block: Improve compatibility with Gutenberg 10.1.0.
```

File: projects/plugins/jetpack/changelog/backfill-pr-19053 [hunk 15]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+AMP: Fix compatibility with Gravatar Hovercards for Pingbacks and Trackbacks.
```
Comment (jeherve) [inline projects/plugins/jetpack/changelog/backfill-pr-19053:4]:
```suggestion
Gravatar Hovercards: Fix compatibility with the AMP plugin for Pingbacks and Trackbacks.
```

**Review by jeherve — DISMISSED** [review]

File: projects/plugins/jetpack/changelog/backfill-pr-18765 [hunk 16]
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: compat.
```
Comment (kraftbj) [inline projects/plugins/jetpack/changelog/backfill-pr-18765:2]:
```suggestion
Type: compat
```

**Review by kraftbj — DISMISSED** [review]

**Comment by kraftbj** [comment]
To confirm, once this is merged, we can delete the "{Has|Needs} Changelog" labels, correct?

**Comment by anomiex** [comment]
> To confirm, once this is merged, we can delete the "{Has|Needs} Changelog" labels, correct?

Yeah, I'd think so. All the "Needs Changelog" have already been removed from tasks as I put them in this PR.

**Review by kraftbj — APPROVED** [review]

**Pull request merged**